tasks/run-matlab-build/v1/buildtool.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright 2024 The MathWorks, Inc.
+
+export interface IRunBuildOptions {
+    Tasks?: string;
```

Comment:
Has Kapil added `BuildOptions` support to Azure yet? I know he's been working his way through the platforms

Reply:
Not yet